examples/object_classification/object_classification.cpp

```diff
@@ -0,0 +1,45 @@
+#include <mlpack/core.hpp>
```

Comment:
I think it would be useful to add some comments to the top of this file describing what it's for, the type of model that it trains, how one might use it, etc.